CorsixTH/Lua/objects/plant.lua

```diff
@@ -67,6 +67,13 @@ object.orientations = {
   },
 }
 
+local anim_mgr = TheApp.animation_manager
+anim_mgr:setStaffMarker(1972,
+    0, {-1, 1, "px"}, 12, {-1, 1, "px"}, 13, {4, -1, "px"})
+anim_mgr:setStaffMarker(1974,
+    0, {-1, 1, "px"}, 19, {-1, 1, "px"}, 20, {2, -1, "px"}, 21, {6, 0, "px"})
```

Comment:
Again a little bit too magic imo.

Reply:
The large text above the set*Marker functions explains it. I am not sure what you would expect here.

Reply:
If you have suggestions, I'd like to hear about them.

I am not too happy with the current form recognition code inside the `set*Marker` functions in recognizing the form, but that's what exists.

Also not sure how to avoid duplicating the animation number.

Last but not least, you get marker positions for multiple frames within a single animation. This marker position data is thus way larger than the single animation number.  
This is not entirely true, often you have groups of animations that share the same set marker positions. (Eg different patient types doing the same thing.)

Reply:
I think where other calls label `kf1`, `kf2` etc beforehand that was ok enough for now. That's all I'm suggesting following at the moment (for both new occurrences in the patch)

Reply:
I think the main reason for having those was to save memory (less table instances), but sure I can generate such output.